components/Currency/CurrencyPrice/CurrencyPrice.tsx

```diff
@@ -33,14 +33,30 @@ export const CurrencyPrice: FC<CurrencyPriceProps> = ({
 	formatOptions,
 	...rest
 }) => {
+	const cleanPrice = useMemo(() => {
+		try {
+			return wei(price);
+		} catch {
+			return '';
```

Comment:
The type is `WeiSource` which can be string, number, or wei but _shouldn't_ be null. In this case since it will be converted to wei, I think an empty string works to force the value to something other than 0 in those cases.

The cases where this fails are strings, so this forces bad strings that can't be converted (`'adam'`) to something that can be converted (`''`)

Reply:
ok makes sense, all good then